src/plugins/stacked-barchart/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require("./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json"),
+};
```

Comment:
Move this file to `src/plugins`? Should we consider working in `/src/plugins`?

Reply:
Unfortunately, `grafana-toolkit plugin:build` automaticaly re-generates the `.prettierrc.js` and `jest.config` and it seems there  is no option to disable this.